src/ui/server/controllers/insurance/eligibility/buyer-country/index.ts

```diff
@@ -19,104 +20,114 @@ export const PAGE_VARIABLES = {
 export const TEMPLATE = TEMPLATES.SHARED_PAGES.BUYER_COUNTRY;
 
 export const get = async (req: Request, res: Response) => {
-  if (!req.session.submittedData || !req.session.submittedData.insuranceEligibility) {
-    req.session.submittedData = {
-      ...req.session.submittedData,
-      insuranceEligibility: {},
-    };
-  }
-
-  const countries = await api.external.getCountries();
-
-  if (!countries || !countries.length) {
-    return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
-  }
-
-  let countryValue;
-
-  if (req.session.submittedData && req.session.submittedData.insuranceEligibility[FIELD_IDS.BUYER_COUNTRY]) {
-    countryValue = req.session.submittedData.insuranceEligibility[FIELD_IDS.BUYER_COUNTRY];
-  }
+  try {
+    if (!req.session.submittedData || !req.session.submittedData.insuranceEligibility) {
```

Comment:
can we destructure or will it break?

Reply:
Can we look into this separately, we can probably destructure a few bits.


These things haven't changed in this PR, just the indentation because of wrapping in a try/catch